Quick-Start.md

```diff
@@ -0,0 +1,203 @@
+needs to be some cloud infrastructure on the backend for users to get the benefit of using the SDK. Currently, we support the CodeFlare stack.
+
+This stack integrates well with Red Hat OpenShift AI and [Open Data Hub](https://opendatahub.io/), and helps to bring batch workloads, jobs, and queuing to the Data Science platform. Although this document will guide you through setting up with Red Hat OpenShift AI (RHOAI), the steps are also applicable if you are using Open Data Hub (ODH). Both platforms are available in the OperatorHub, and the installation and configuration steps are quite similar. This guide will proceed with RHOAI, but feel free to apply the instructions to ODH as needed.
+
+## Prerequisites
+
+### Resources
+
+In addition to the resources required by default Red Hat OpenShift AI deployments, you will need the following to deploy the Distributed Workloads stack infrastructure pods:
+
+```text
+Total:
+    CPU: 1600m (1.6 vCPU)
+    Memory: 2048Mi (2 GiB)
+```
+
+> [!NOTE]
+> The above resources are just for the infrastructure pods. To be able to run actual
+> workloads on your cluster you will need additional resources based on the size
+> and type of workload.
+
+### GPU Worker Node
+
+> [!IMPORTANT]
+> This step is necessary only if you require GPU capabilities for your workloads and your OpenShift cluster does not already include GPU-equipped nodes, follow these steps:
+
+1. **Open the OpenShift Cluster Console.**
+2. Navigate to **your-cluster** -> **Machine pools**.
+3. Click on **“Add machine pool”**.
+4. Provide a **name** for the new machine pool.
+5. In the **“Compute node instance type”** dropdown, scroll all the way down and search for the GPU instance type `g4dn.xlarge` or similar.
+6. Click on **Add machine pool** to finalize the creation of your new GPU-enabled machine pool.
+
+After adding the machine pool, OpenShift will begin provisioning the new GPU worker
+node. This process can take a few minutes. Once completed, the new node will be ready to handle GPU-accelerated workloads.
+
+> [!NOTE]
+> The `g4dn.xlarge` instance type is used for GPU worker nodes. Ensure this instance type meets your application needs or select another as required.
+
+## Automatic deployment for developers
+
+As a quick alternative to the following manual deployment steps an automatic _Makefile_
+script can be used to deploy the CodeFlare stack. This script also deploys the prerequisite
+operators and the entire CodeFlare stack up to the step
+[Configure Kueue for Task Scheduling](#configure-kueue-for-task-scheduling).
+This will give you a fresh install of all the components. To use this script,
+clone the repo and execute:
+
+1. Clone the Repository
+
+```bash
+git clone https://github.com/project-codeflare/codeflare-operator.git
+cd codeflare-operator
+```
+
+2. Run the Makefile script
+
+```bash
+make all-in-one
+```
+
+> [!TIP]
+> Execute `make help` to list additional available operations.
+
+## Red Hat OpenShift AI
```

Comment:
I'm not sure if it is clear that this is an alternative to using `make all-in-one` rather than further instructions